tests/integration/standard/test_metadata.py

```diff
@@ -1204,7 +1204,8 @@ def test_export_keyspace_schema_udts(self):
         cluster.shutdown()
 
     @greaterthancass21
-    @pytest.mark.xfail(reason='Column name in CREATE INDEX is not quoted. It\'s a bug in driver or in Scylla')
+    @xfail_scylla_version(reason='Column name in CREATE INDEX is not quoted. It\'s a bug in driver or in Scylla',
+                          oss_scylla_version="5.2", ent_scylla_version="2023.1")
```

Comment:
If this is fixed in 5.2, is there an issue / PR numbers? If so, please include them in the message.